projects/storefrontlib/src/lib/ui/layout/config/default-layout-config.ts

```diff
@@ -18,6 +18,20 @@ export const defaultLayoutConfig: LayoutConfig = {
         'Section4',
         'Section5'
       ]
+    },
+    CategoryPageTemplate: {
+      slots: ['Section1', 'Section2', 'Section3']
```

Comment:
In the removed file `category-page-layout.component.html` there was also the `Section4`. Was it not included here on purpose?